app/model/Payment/Payment.php

```diff
@@ -115,6 +115,13 @@ class Payment extends Aggregate
      */
     private $closedAt;
 
+    /**
+     * @ORM\Column(type="string", length=64, nullable=true)
+     *
+     * @var string|NULL
+     */
+    private $closedBy;
```

Comment:
Můžeme do toho názvu nějak dostat, že se jedná o uživatele? (resp. jméno uživatele?)

Reply:
Upravim to na closedByUsername